docker-host/main.go

```diff
@@ -0,0 +1,140 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/client"
+	"github.com/robfig/cron/v3"
+	machineryvars "github.com/uselagoon/machinery/utils/variables"
+)
+
+const EnvOverrideHost = "DOCKER_HOST"
+
+var dockerHost = machineryvars.GetEnv("DOCKER_HOST", "docker-host")
+var repo = machineryvars.GetEnv("REPOSITORY_TO_UPDATE", "amazeeio")
+var REGISTRY = machineryvars.GetEnv("REGISTRY", "docker-registry.default.svc:5000")
+var BIP = machineryvars.GetEnv("BIP", "172.16.0.1/16")
+var REGISTRY_MIRROR = machineryvars.GetEnv("REGISTRY_MIRROR", "https://imagecache.amazeeio.cloud")
```

Comment:
I'd like to see this set to `""` as the default rather than using our imagecache domain. We plan to deprecate having this as a default, and only defining it it in helm (you may need to update the helmchart to add this as a new settable option if it doesn't already exist)